### ISSUE TITLE
Alt text appears as caption and image title fails to appear in News Sitemap

### ISSUE THREAD

**Issue by Pcosta88** [Yoast/wpseo-news#327]
### What did you expect to happen?
The caption to appear in the caption tags. The title to appear.
### What happened instead?
It did not
### How can we reproduce this behavior?
**1.Create a post**

**2.Add an image via Media Folder and set the image title, caption, and title.**

![screen shot 2017-11-15 at 1 32 35 pm](https://user-images.githubusercontent.com/8238074/32853563-b082ac54-ca09-11e7-9214-5af3f44c5d94.png)

**3.See image embed**

![screen shot 2017-11-15 at 1 32 41 pm](https://user-images.githubusercontent.com/8238074/32853577-b831c9d0-ca09-11e7-8d2f-df3483d67470.png)

**4.Publish it and go to news sitemap. View source code. See alt-text appear as caption. Title Does not appear.**

![screen shot 2017-11-15 at 1 32 50 pm](https://user-images.githubusercontent.com/8238074/32853597-c761fa42-ca09-11e7-99d6-d82368b83de9.png)

**5.Click on image and add an image title attribute**

![screen shot 2017-11-15 at 1 35 21 pm](https://user-images.githubusercontent.com/8238074/32853637-e448af5c-ca09-11e7-9e32-2aee11455fde.png)

**6.Save page. Go to News Sitemap. View Source code. See image title attribute outputted as image title.**

![screen shot 2017-11-15 at 1 37 15 pm](https://user-images.githubusercontent.com/8238074/32853709-235f5718-ca0a-11e7-97f1-bcc7dbb11636.png)

### Technical info
* WordPress version:4.8.3
* News 5.8 and Yoast 5.8

**Comment by CarolineGeven** [comment]
This does not seem as easy as it seems at first glance. 

When adding an image to a post, you can set the caption. However, if you update the caption through the post, the caption of the attachment itself, does not get updated. You can, with a workaround, get the caption of the attachment, but that would be of no use if it is not the same as the image caption set in the post.
